projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/write-linked-cases-field.component.ts

```diff
@@ -102,6 +101,13 @@ export class WriteLinkedCasesFieldComponent extends AbstractFieldWriteComponent
     });
   }
 
+  getOrgService(): void {
+    const servicesApiUrl = `refdata/location/orgServices?ccdCaseType=${this.caseDetails.case_type.id}`;
+    this.commonDataService.getServiceOrgData(servicesApiUrl).subscribe(result => {
+      this.getLinkedCaseReasons(result[0].service_id);
```

Comment:
Unfortunately you need to handle each of the case types returned. A single case type can contain cases from more than one service. Civil do this, I wish they hadn't but we can't just take the first

Reply:
@andywilkinshmcts I require more clarification from you on this. Better to chat over the phone